### PR TITLE
aa/update README.md

### PR DESCRIPTION
**Pull request by alexisintech**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,26 +2,28 @@
...
+## Community
 
-### I have feedback
+If you have questions or just need a little clarity, our community and our team is ready to help you in the official [Clerk discord](https://discord.com/invite/b5rXHjAg7A).
 
-Feel free to create an issue with the feedback label. We will take the feedback and iterate on it.
+## Found an issue in the docs?
 
-### How do I contribute?
+If you have found an issue with our documentation, please create an [issue](https://github.com/clerkinc/clerk-docs/issues) or create a [pull request](https://github.com/clerkinc/clerk-docs/pulls) to fix it.
```
Comment (BRKalow) [inline README.md:25]:
```suggestion
If you have found an issue with our documentation, please create an [issue](https://github.com/clerkinc/clerk-docs/issues) or a [pull request](https://github.com/clerkinc/clerk-docs/pulls) to fix it.
```

File: README.md [hunk 2]
```diff
@@ -2,26 +2,35 @@
   <a href="https://clerk.dev?utm_source=github&utm_medium=clerk_docs" target="_blank" rel="noopener noreferrer">
     <picture>
       <source media="(prefers-color-scheme: dark)" srcset="https://images.clerk.dev/static/logo-dark-mode-400x400.png">
-      <img src="https://images.clerk.dev/static/logo-light-mode-400x400.png" height="64">
+      <img src="https://images.clerk.dev/static/logo-light-mode-400x400.png" height="120">
```
Comment (BRKalow) [inline README.md:5]: It looks like most of our other READMEs use the same size logo, should we keep it the smaller size for consistency?

ref:
* https://github.com/clerkinc/javascript/blob/main/README.md

File: README.md [hunk 3]
```diff
@@ -2,26 +2,35 @@
...
-# Clerk Documentation
+<h1 align="center">
+  Clerk Documentation
+</h1>
 
-This repository contains the content for [Clerk's documentation](https://clerk.com/docs).
+Clerk is more than authentication, and we want to convey every feature of Clerk's as best as we can to make authentication and user management simpler for everyone. Explaining these concepts can sometimes be tricky, but with the help of a community, we can collaborate, share ideas, find the right words, and achieve greater clarity.
 
-### I've found an issue with your documentation
+We encourage you to contribute to Clerk's documentation! You can get started by following [this guide](./CONTRIBUTING.md).
```
Comment (BRKalow) [inline README.md:17]: I think I understand what the intent is here, but I would expect this content to live under a "Contributing" or "Community" section, not necessarily the top-level blurb.

File: README.md [hunk 4]
```diff
@@ -2,26 +2,35 @@
   <a href="https://clerk.dev?utm_source=github&utm_medium=clerk_docs" target="_blank" rel="noopener noreferrer">
```
Comment (BRKalow) [inline README.md:2]: We should update this URL to be `clerk.com` 👀

File: README.md [hunk 5]
```diff
@@ -2,26 +2,35 @@
...
+## Want to leave feedback?
 
-See the [contributing](./CONTRIBUTING.md) document for more information on how to successfully contribute to Clerk's documentation.
+Feel free to create an [issue](https://github.com/clerkinc/clerk-docs/issues) with the **feedback** label. Our team will take a look at it and get back to you as soon as we can!
+
+## Connect with us
+
+You can discuss ideas, ask questions, and meet others from the community in our [Discord](https://discord.com/invite/b5rXHjAg7A). 
+
+If you prefer, you can also find support through our [Twitter](https://twitter.com/ClerkDev), or you can [email](mailto:support@clerk.dev) us!
```
Comment (BRKalow) [inline README.md:36]: We might want to check with @perkinsjr to see if encouraging people to use Twitter for support is something we want to recommend. 💭

File: README.md [hunk 6]
```diff
@@ -2,26 +2,35 @@
...
 
-### I have feedback
+## Found an issue?
 
-Feel free to create an issue with the feedback label. We will take the feedback and iterate on it.
+If you have found an issue with our documentation, please create an [issue](https://github.com/clerkinc/clerk-docs/issues).
 
-### How do I contribute?
+If it's a quick fix, such as a misspelled word or a broken link, feel free to skip creating an issue.  
+Go ahead and create a [pull request](https://github.com/clerkinc/clerk-docs/pulls) with the solution :rocket:
```
Comment (BRKalow) [inline README.md:26]:
```suggestion
Go ahead and create a [pull request](https://github.com/clerkinc/clerk-docs/pulls) with the solution. :rocket:
```

File: README.md [hunk 7]
```diff
@@ -2,26 +2,35 @@
...
   <br />
 </p>
 
-# Clerk Documentation
+<h1 align="center">
+  Clerk Documentation
+</h1>
 
-This repository contains the content for [Clerk's documentation](https://clerk.com/docs).
+Clerk is more than authentication, and we want to convey every feature of Clerk's as best as we can to make authentication and user management simpler for everyone. Explaining these concepts can sometimes be tricky, but with the help of a community, we can collaborate, share ideas, find the right words, and achieve greater clarity.
```
Comment (BRKalow) [inline README.md:15]:
```suggestion
Clerk is more than authentication, and we want to convey every feature of Clerk as best as we can to make authentication and user management simpler for everyone. Explaining these concepts can sometimes be tricky, but with the help of a community, we can collaborate, share ideas, find the right words, and achieve greater clarity.
```

**Review by BRKalow — COMMENTED** [review]
Looking good ✨

File: README.md (hunk 3 above)
Comment (alexisintech) [reply README.md:17]: Hmm, I was kind of taking after other docs like Astro, Solid, etc but we can simplify.

File: README.md (hunk 5 above)
Comment (alexisintech) [reply README.md:36]: that's actually straight from our /support page

Comment (BRKalow) [reply README.md:36]: Ah, nice! That's validation enough for me then.

File: README.md (hunk 3 above)
Comment (BRKalow) [reply README.md:17]: Ah, I see. I like how Astro does it! With the separate sections:

* https://github.com/withastro/docs#we-are-astro
* https://github.com/withastro/docs#you-are-awesome

**Review by BRKalow — APPROVED** [review]
👏

**Pull request merged**